### PR TITLE
Fix/FE adjust most popular movies component according to design 1440px  and link to movies details

### PR DESCRIPTION
**Pull request by Kositthai**
# Description

- Adjust most popular movies component according to design 1440px 
- Link most popular element to movies/id when click on it 

- Edited return error to Throw error in MostPopular file 

# Jira task link
https://hackyourfuture-dk.atlassian.net/jira/software/projects/LANDING/boards/6?selectedIssue=LANDING-69
https://hackyourfuture-dk.atlassian.net/jira/software/projects/LANDING/boards/6?selectedIssue=LANDING-58

# Screenshots (if applicable)

https://github.com/HackYourFuture-CPH/CINEMANIA/assets/94117213/aac7c637-5a5a-4443-a121-a4ce05832b1f

### CODE REVIEW

File: packages/client/src/containers/CarouselWrapper/MostPopular.jsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const MostPoppular = () => {
         const data = await response.json();
         setPopularMovies(data.slice(1, 12));
```
Comment (jawkha) [inline packages/client/src/containers/CarouselWrapper/MostPopular.jsx:13]: @Kositthai Can you please confirm the structure of the `data` object here? I am getting an error here.

File: packages/client/src/containers/CarouselWrapper/MostPopular.jsx
Comment (jawkha) [inline packages/client/src/containers/CarouselWrapper/MostPopular.jsx:1]: @Kositthai It looks like the name of the component has a typo. Can you please also take a look at it?

**Review by jawkha — CHANGES_REQUESTED** [review]
Other than the two minor issues, it all looks quite good.
Really good work!!

**Comment by Kositthai** [comment]
> Other than the two minor issues, it all looks quite good. Really good work!!

Thank you so much for your review! I have fixed my code as your suggestion :) 
Kindly let me now if I need to change anything else otherwise kindly approve

**Review by jawkha — APPROVED** [review]
Seems to be working now. Good work!!

**Comment by Kositthai** [comment]
> Seems to be working now. Good work!!

Thank you so much!

**Pull request merged**